bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeperAdmin.java

```diff
@@ -95,6 +96,7 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(BookKeeperAdmin.class);
     private static final Logger VERBOSE = LoggerFactory.getLogger("verbose");
+    public static final BiConsumer<Long, Long> NOOP_BICONSUMER = (l, e) -> { };
```

Comment:
nit: this one should be private

Reply:
this is used in testcases as well, so I changed it from private to public.

Reply:
I prefer not to add such kind of generic constants to a public API (BookKeeperAdmin is used on downstream projects).

can you please move it in some Utility class in tests ?

Reply:
Hey @eolivelli I made it private as you suggested.